loki/README.md

```diff
@@ -0,0 +1,128 @@
+# Installing a custom Loki stack in Kyma
+
+## Overview
+
+The Kyma Loki component brings limited configuration options in contrast to the upstream [loki-stack](https://github.com/grafana/helm-charts/tree/main/charts/loki-stack) chart. Modifications might be reset at next upgrade cycle.
+
+An alternative can be a parallel installation of the upstream chart offering all customization options. The following instructions outline how to achieve such installation in co-existence to the Kyma stack.
+
+>**CAUTION:** This example uses the Grafana Loki version, which is distributed under AGPL-3.0 only and might not be free of charge for commercial usage.
+
+>**CAUTION:** These instructions install Loki in a lightweight setup that does not fulfil production-grade qualities. Consider using a scalable setup based on an object storage backend instead (see [Simple scalable deployment of Grafana Loki with Helm](https://grafana.com/docs/loki/latest/installation/simple-scalable-helm/)).
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+- Kubectl version 1.22.x or higher
+- Helm 3.x
+
+## Installation
+
+### Preparation
+
+1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+
+    ```bash
+    export KYMA_LOKI_EXAMPLE_NS="{namespace}"
+    ```
+
+2. Export the Helm release name that you want to use. It can be any name, but be aware that all resources in the cluster will be prefixed with that name. Replace the `{release-name}` placeholder in the following command and run it:
+    ```bash
+    export HELM_RELEASE_NAME="{release-name}"
+    ```
+
+3. Update your Helm installation with the required Helm repository:
+
+    ```bash
+    helm repo add grafana https://grafana.github.io/helm-charts
+    helm repo update
+    ```
+
+### Install the Loki stack
+
+Run the Helm upgrade command, which installs the chart if not present yet.
+```bash
+helm upgrade --install --create-namespace -n ${KYMA_LOKI_EXAMPLE_NS} ${HELM_RELEASE_NAME} grafana/loki-stack -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/loki-values.yaml --set promtail.enabled=false --set grafana.enabled=false
+```
+
+You can either use the [loki-values.yaml](./loki-values.yaml) provided in this `loki` folder, which contains customized settings deviating from the default settings, or create your own `values.yaml` file.
+
+### Verify the installation
+
+Check that the `loki` Pod has been created in the Namespace and is in the `Running` state:
+
+```bash
+kubectl -n ${KYMA_LOKI_EXAMPLE_NS} get pod ${HELM_RELEASE_NAME}-0
+```
+
+### Activate log shipment using a LogPipeline
+
+1. Download the [logpipeline](https://raw.githubusercontent.com/kyma-project/examples/main/loki/logpipeline-custom.yaml) and replace the `{release-name}` and `{namespace}` placeholder.
```

Comment:
This link seems broken: 

<img width="871" alt="image" src="https://user-images.githubusercontent.com/76950046/197816688-92b333bc-62d9-4e66-a9c0-a43d596a8ffb.png">

Reply:
Yes it doesn't exist yet because it will be introduced with this PR (see files). Don't know how we usually handle this, but I temporary removed the link and will create another PR after this one to introduce it.

Reply:
aaaah 🤦 
Thank you!